lib/CXGN/Trial/ParseUpload/Plugin/TrialExcelFormat.pm

```diff
@@ -312,7 +312,7 @@ sub _validate_with_plugin {
     foreach my $treatment_name (@treatment_names){
         if($worksheet->get_cell($row,$treatment_col)){
             my $apply_treatment = $worksheet->get_cell($row,$treatment_col)->value();
-            if (defined($apply_treatment) && $apply_treatment ne '1'){
+            if ( ($apply_treament ne '' ) && defined($apply_treatment) && $apply_treatment ne '1'){
```

Comment:
there is a typo here in apply_treament vs apply_treatment @lukasmueller

Reply:
Great catch, thanks! Fixed.